openmmtools/states.py

```diff
@@ -1702,6 +1702,11 @@ def _read_context_state(self, context, check_consistency):
 # COMPOUND THERMODYNAMIC STATE
 # =============================================================================
 
+class ComposableStateError(Exception):
+    """Error raised by a ComposableState."""
+    pass
+
+
 class IComposableState(utils.SubhookedABCMeta):
```

Comment:
`I` usually stands for `Interface`. It's not standard but very common: http://stackoverflow.com/questions/10723839/python-naming-conventions-for-interfaces-and-abstract-classes